seatunnel-api/src/main/java/org/apache/seatunnel/api/configuration/util/OptionRule.java

```diff
@@ -242,12 +242,21 @@ private void verifyDuplicateWithOptionOptions(
         }
 
         private void verifyRequiredOptionDuplicate(@NonNull RequiredOption requiredOption) {
+            verifyRequiredOptionDuplicate(requiredOption, false);
+        }
+
+        private void verifyRequiredOptionDuplicate(
+                @NonNull RequiredOption requiredOption,
+                @NonNull Boolean ignoreVerifyDuplicateWithOptionOptions) {
```

Comment:
Could you add some comment about when should set ignore to `false` or `true`?

Reply:
Get, wait for add comments